envsec/internal/jetcloud/git.go

```diff
@@ -0,0 +1,31 @@
+package jetcloud
+
+import (
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+func createGitIgnore(wd string) error {
+	gitIgnorePath := filepath.Join(wd, dirName, ".gitignore")
+	return os.WriteFile(gitIgnorePath, []byte("*"), 0600)
+}
+
+func gitRepoURL(wd string) (string, error) {
+	cmd := exec.Command("git", "config", "--get", "remote.origin.url")
```

Comment:
Is this going to be different for `ssh` vs `https` method, based on how the user cloned the repo?

If so, are we doing the parsing/unification in the api?